core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -504,7 +504,7 @@ class DAGScheduler(
       callSite: CallSite,
       allowLocal: Boolean,
       resultHandler: (Int, U) => Unit,
-      properties: Properties = null)
```

Comment:
Is this ever called without supplying a value for `properties`?  Can we remove the default value here?